### PR TITLE
build(Nightly run stryker): Update the stryker dashboard every night

### LINKED ISSUE #104: Use stryker on the stryker build
It would be cool if we use stryker-net on stryker-net itself in the build. That way we can check our own score.

### PR DESCRIPTION
**Pull request by richardwerkman**
closes #104

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -186,6 +186,38 @@ stages:
...
+        arguments: '--reporters "['dashboard', 'dots']" --dashboard-project github.com/stryker-mutator/stryker-net --dashboard-version master --dashboard-module cli'
+        workingDirectory: 'src\Stryker.CLI\Stryker.CLI.UnitTest'
+  - job: StrykerCore
+    displayName: Run Stryker.Core
+    - task: DotNetCoreCLI@2
+      displayName: Run Stryker.NET
+      inputs:
+        command: 'custom'
+        custom: 'stryker'
+        arguments: '--reporters "['dashboard', 'dots']" --dashboard-project github.com/stryker-mutator/stryker-net --dashboard-version master --dashboard-module cli'
```
Comment (Mobrockers) [inline azure-pipelines.yml]: Module should be core

**Comment by richardwerkman** [comment]
@Mobrockers I removed the condition because I want it to run in this PR to see if it works. Once it works I'll add the condition again :+1:

**Comment by Mobrockers** [comment]
Perhaps it makes sense to run this with the sonar analysis (but only on timed trigger) and make that a more generic check code quality stage?

**Pull request merged**